### PR TITLE
fix names

### PR DESCRIPTION
**Pull request by pedrofurst**
* fixed function names

refs #23

### CODE REVIEW

**Comment by cacontinuos** [comment]
By analyzing the blame information on this pull request, we identified @fernahh and @caarlos0 to be potential reviewers

**Comment by VictorBogo** [comment]
+1

**Comment by cacontinuos** [comment]
-1

PR should have a **good** description!

**Comment by matheuspoleza** [comment]
++

File: index.html [hunk 1]
```diff
@@ -80,7 +80,7 @@ <h2 class="semi-bold-title">Usage</h2>
             <tbody>
               <tr>
                 <td class="col1"></td>
-                <td class="col34">CreditCard#isValidateExpirationDate()</td>
```
Comment (caarlos0) [inline index.html]: it will be better if it was named `isExpirationDateValid`

File: index.html [hunk 2]
```diff
@@ -92,7 +92,7 @@ <h2 class="semi-bold-title">Usage</h2>
               </tr>
               <tr>
                 <td class="col1"></td>
-                <td class="col34">CreditCard#isValidateSecuryCode()</td>
+                <td class="col34">CreditCard#isValidSecurityCode()</td>
```
Comment (caarlos0) [inline index.html]: here too, `isSecurityCodeValid`

Comment (pedrofurst) [inline index.html]: ++

**Pull request merged**